src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -41,7 +59,13 @@ object Boot extends App {
 
     dataSource.inTransaction() { txn => txn.withGraph { graph => VertexSchema.createVertexClasses(graph.asInstanceOf[OrientGraph]) } }
 
-    val slickDataSource = DataSource(DatabaseConfig.forConfig[JdbcProfile]("database", conf))
+    val slickDbConf = DatabaseConfig.forConfig[JdbcProfile]("slick", conf)
```

Comment:
I like "slick" instead of "database" because some of the config is slick-specific.